packages/artillery/lib/cmds/run-lambda.js

```diff
@@ -60,6 +64,10 @@ RunLambdaCommand.flags = {
     // locally defaults to number of CPUs with mode = distribute
     default: '1'
   }),
+  container: Flags.boolean({
+    description: 'Run the test in a container (recommended)',
```

Comment:
```suggestion
    description: 'Run the test in a container (experimental)',
```

Reply:
```suggestion
    description: 'Use a container image for Lambda (experimental)',
```

Think we need to be more specific here than "run the test in a container". Lambda users will probably be aware of the two different packaging methods.